ui/desktop/electron-app/yarn.lock

```diff
@@ -2080,10 +2080,10 @@ jsprim@^1.2.2:
     json-schema "0.2.3"
     verror "1.10.0"
 
-jszip@^3.1.0:
-  version "3.6.0"
-  resolved "https://registry.yarnpkg.com/jszip/-/jszip-3.6.0.tgz#839b72812e3f97819cc13ac4134ffced95dd6af9"
-  integrity sha512-jgnQoG9LKnWO3mnVNBnfhkh0QknICd1FGSrXcgrl67zioyJ4wgx25o9ZqwNtrROSflGBCGYnJfjrIyRIby1OoQ==
+jszip@^3.1.0, jszip@^3.7.0:
```

Comment:
Ups... that is my fault. I have merged change on `package.json` without running `yarn install` and commit the `.lock` file. This is expected and ok, I have a PR pending with this change, so who ever merges first will get it done :)